Automattic/jetpack
# Social: Add the basic Open Graph tags

Without the `og:title` tag, the post doesn't pass validation for twitter and the card/featured image is not used. It also generates warnings on Facebook.

#### Changes proposed in this Pull Request:
This PR adds title, description and URL tags. It's mainly a copy of the code from `projects/plugins/jetpack/functions.opengraph.php`, which is a temporary measure until we refactor it to form a package.

#### Jetpack product discussion
p1HpG7-g6I-p2#comment-54679

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
- Create a site with Jetpack Social enabled and connected
- Publish a post with a featured image
- Put the URL into https://cards-dev.twitter.com/validator
- With this PR. Check that the card validates and that it displays as expected.

N.B. It can take two requests for the featured image to display.
